src/lib/dev-environment/dev-environment-cli.js

```diff
@@ -74,7 +74,7 @@ declare function promptForComponent( component: string, allowLocal: boolean, def
 
 export async function handleCLIException( exception: Error, trackKey?: string, trackBaseInfo?: any = {} ) {
 	const errorPrefix = chalk.red( 'Error:' );
-	if ( exception instanceof UserError ) {
+	if ( exception instanceof UserError && DEV_ENVIRONMENT_NOT_FOUND !== exception.message ) {
```

Comment:
What about just moving up line 80 `} else if ( DEV_ENVIRONMENT_NOT_FOUND === exception.message ) {` to be the first check?

Reply:
Done